tests/failpoints/cases/test_coprocessor.rs

```diff
@@ -58,14 +58,16 @@ fn test_deadline_3() {
     fail::cfg("kv_cursor_seek", "sleep(2000)").unwrap();
     fail::cfg("copr_batch_initial_size", "return(1)").unwrap();
     let cop_resp = handle_request(&endpoint, req);
+    println!("{:?}", cop_resp);
```

Comment:
should remove this

Reply:
Sorry, I will remove it in my recent PR.